Makefile

```diff
@@ -304,10 +304,18 @@ e2e-test:
 		cargo test test::$(TEST_CASE) -- --color always --exact --nocapture --test-threads=1
 
 .PHONY: e2e-tests
-e2e-tests: # Run e2e tests
+e2e-tests: # Run cross-chain transfer e2e tests. All tests must be run without interruption, the order is important.
```

Comment:
Lest add an info here what the e2e-tests requires to be run. Also add section to the project readme how to run e2e tests locally.

Reply:
done